.github/workflows/npmsmoketest.yml

```diff
@@ -58,22 +62,16 @@ jobs:
             nohup verdaccio --config .config/verdaccio/config.yaml &>$tmp_registry_log &
 
 
-        - name: Cache Node modules
-          uses: actions/cache@v2
-          with:
-            path: '**/node_modules'
-            key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
-
-
         - name: Publish packages
-          run  : |
+          run: |
             cd main
             yarn install
             npm publish --registry ${{env.LOCAL_REGISTRY}} packages/css-framework
```

Comment:
Is it worth setting the NPM registry here globally so you don't have to explicitly provide the `--registry` argument every time?

```
npm config set registry ${{env.LOCAL_REGISTRY}}
```

Reply:
We could although in the next job we need to run yarn install again (not using local registry), so would need to set & unset

Reply:
Ok fair enough - feel free to just leave it as is then